boilerplate/app/screens/DemoPodcastListScreen.tsx

```diff
@@ -27,7 +30,7 @@ export function DemoPodcastListScreen(props: DemoTabScreenProps<"DemoPodcastList
   return (
     <Screen preset="fixed" safeAreaEdges={["top"]}>
       <FlatList<Episode>
-        data={episodeStore.episodes}
+        data={episodeStore.episodes.slice()}
```

Comment:
@robinheinze Do we need `slice` here after all?

Reply:
No i don't think we do. Updated.